image-builder/README.md

```diff
@@ -14,12 +14,97 @@ We use the Docker's building system and convert it at the end to a disk image th
 [More info](https://github.com/scaleway/image-tools)
 
 
-:warning: **Documentation incoming** :warning:
+## How to build a custom image (using [scw](https://github.com/scaleway/scaleway-cli))
+
+**My custom image's description**
+- based on [ubuntu vivid](https://github.com/scaleway/image-ubuntu)
+- `cowsay` pre-installed
+
+---
+
+##### 1 Making the environment
+
+```bash
+root@yourmachine> scw run --rm --name="buildcowsay" builder
+...SKIP
+*****************************************************************************
+
+   Welcome on image-builder.
+   Here, you will be able to do your own images.
+
+   To configure your environment please run:
+
+   $> image-builder-configure
+
+*****************************************************************************
+...SKIP
+root@buildcowsay> image-builder-configure
+Login (cloud.scaleway.com):                     # yourmail
+Password:                                       # yourpassword
+Do you want to upload an SSH key ?
+[0] I don t want to upload a key !
+[1] id_rsa.pub
+Which [id]: 1                                   # you must upload your local key
```

Comment:
Not sure you need to upload your local key, since you are already connected to this builder using `scw exec` -> your real key is already on your account, and this builder will use the `--env="AUTHORIZED_KEY"` option to send local key